CardPhunTests/CardTests/CardSetTest.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Linq;
+using CardPhun;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CardPhunTests.CardTests
+{
+    [TestClass]
+    public class CardSetTest
+    {
+        [TestMethod]
+        public void AddToSetCountTest()
+        {
+            var card = new TestCard(1, Znak.CLUBS);
+            var cardSet = new TestCardSet();
+            cardSet.AddToSet(card);
+            Assert.AreEqual(cardSet.Count, 1);
+        }
+
+        [TestMethod]
+        public void AddToSetNumberTest()
+        {
+            var card = new TestCard(1, Znak.CLUBS);
+            var cardSet = new TestCardSet();
+            cardSet.AddToSet(card);
+            Assert.AreEqual(cardSet.GetSumOfCards(), 1);//da li sam ovim proverio getsumofcards, ili to moram posebno kada dodjem do toga?
```

Comment:
Ovo je preeevise induktivno. Dodaj recimo 1,2,3,6,8 i proveri da je zbir dobar. Ako hoces da budes fancy, random izgenerisi 5-6 karata, saberi ih rucno, i onda to uporedi sa tim sto ti vrati GetSumOfCards().